.gitignore

```diff
@@ -9,6 +9,7 @@ Publish
 *.suo
 *.cscfg
 *.Cache
+*.pem
```

Comment:
❓ Oh, did you need to add a pem file?

Reply:
Didn't end up needing to, but I was using a pem earlier so added it to my gitignore so I don't accidentally commit